operator-metadata/manifests/bundle.clusterserviceversion.yaml

```diff
@@ -1415,17 +1426,6 @@ spec:
       type: MultiNamespace
     - supported: true
       type: AllNamespaces
-  relatedImages:
-    - name: strimzi-cluster-operator
-      image: registry.redhat.io/amq-streams/strimzi-rhel9-operator@sha256:06de6eaeb8cae4ea31f05a427bbe53f9d078de0030bd56fce4ecfbb5d0be461a
-    - name: strimzi-kafka-360
```

Comment:
Does the naming in relatedImages matter? Do we know if OSBS will generate the same names?

Reply:
Ok OSBS is going to lowercase the names of the vars so will generate `kafka_image_3_6_0`